gemini/prompts/examples/text_classification.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Change this to use the <code>GenerationConfig</code> class <code>from vertexai.generative_models</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>